flixel/input/touch/FlxTouch.hx

```diff
@@ -38,30 +50,42 @@ class FlxTouch extends FlxPointer implements IFlxDestroyable implements IFlxInpu
 	{
 		input = null;
 		justPressedPosition = FlxDestroyUtil.put(justPressedPosition);
+		_pressure = null;
 		flashPoint = null;
 	}
 
 	/**
-	 * Resets the justPressed/justReleased flags and sets touch to not pressed.
+	 * Resets the justPressed/justReleased flags, sets touch to not pressed and sets touch pressure to 0.
 	 */
-	public function recycle(x:Int, y:Int, pointID:Int):Void
+	public function recycle(x:Int, y:Int, pointID:Int, pressure:Float):Void
 	{
 		setXY(x, y);
 		input.ID = pointID;
 		input.reset();
+		setPressure(pressure);
+		if (pressure > 0)
+			setIsPen(true);
+		else
+			setIsPen(false);
 	}
 
 	/**
 	 * @param	X			stageX touch coordinate
 	 * @param	Y			stageX touch coordinate
 	 * @param	PointID		touchPointID of the touch
+	 * @param	pressure	A value between 0.0 and 1.0 indicating force of the contact with the device. If the device does not support detecting the pressure, the value is 1.0.
 	 */
-	function new(x:Int = 0, y:Int = 0, pointID:Int = 0)
+	function new(x:Int = 0, y:Int = 0, pointID:Int = 0, pressure:Float = 1.0)
 	{
 		super();
 
 		input = new FlxInput(pointID);
 		setXY(x, y);
+		setPressure(pressure);
+		if (pressure > 0)
```

Comment:
Same. Remove the pen property and just set pressure to whatever was passed in from FxTouchManager